forgettable-frontend/src/pages/SignInPage/SignInPage.js

```diff
@@ -4,18 +4,44 @@ import googleLogin from '../../assets/icons/google-login.svg';
 import logoBlack from '../../assets/logos/logo-black.svg';
 import {authentication} from '../../firebase.js';
 import {signInWithPopup, GoogleAuthProvider} from 'firebase/auth';
+import {createUser} from '../../services';
 
 
 export default function SignInPage({setIsLoggedIn}) {
+  async function dealWithUser(user) {
+    // post to the backend
+    const userObj = {
+      first_name: user.displayName,
+      last_name: user.displayName,
```

Comment:
How come these are both displayName?

Reply:
because the display name comes through as "Hazel Williams", and there isnt a first name last name coming through. So we were facing the splitting with space issue again. I can change it if people want tho, i just did this to simplify it for now while we get it out the door.

Reply:
Ahh - just had a read into it, you're right, it doesn't have a split version and there's no proper thing to split it with either D: 
Not too big of a problem IMO, since it's just the user and their name is only ever displayed in the welcome message. Was just curious, haha. In this case could we consider making the user only have a display_name instead? Since there's no way for us to get both anyway :( Not a P1 though, definitely could do that in the next PR or never

Reply:
that would require a change in the backend, but i will make an issue for it if the next group wants to pick it up! not a big deal this time since we dont have any uses for the names rn :) could be a good first issue for them as well